app/views/health/he/search.haml

```diff
@@ -13,6 +13,8 @@
           %em ja sm.
         .o-page__search
           = render 'search_form', tooltip: 'Search with full or partial "first last" or "last, first" for more specific results', data: {placement: 'bottom'}
+          = render_generic_search_form prompt: @prompt, tooltip_title: 'Search with full or partial "first last" or "last, first" for more specific results'
```

Comment:
this was the only page I couldn't figure out how to render, I'm probably missing something

Reply:
I believe you'd need to enable a health emergency via the site config.